big_tests/tests/graphql_last_SUITE.erl

```diff
@@ -290,6 +333,61 @@ admin_logged_user_is_not_old_user_story(Config, _Alice) ->
     Res = admin_list_old_users(null, now_dt_with_offset(100), Config),
     ?assertEqual([], get_ok_value(p(listOldUsers), Res)).
 
+%% Domain admin test cases
+
+domain_admin_try_set_nonexistent_user_last(Config) ->
+    get_unauthorized(admin_set_last(?NONEXISTENT_JID, <<"status">>, null, Config)).
+
+domain_admin_get_nonexistent_user_last(Config) ->
+    get_unauthorized(admin_get_last(?NONEXISTENT_JID, Config)).
+
+domain_admin_try_count_external_domain_active_users(Config) ->
+    get_unauthorized(admin_count_active_users(<<"external-domain.com">>, null, Config)).
```

Comment:
It would be nice to test an existing domain, e.g. `domain_helper:secondary_domain()` - this way we would be sure, that the domain admin is unauthorized to access an existing domain as well.